### ISSUE TITLE
KLD loss question.

### ISSUE THREAD

**Issue by seominseok0429** [mary-phuong/multiexit-distillation#1]
Hi. I am a student in Korea. I was impressed with your iccv2019 announcement.

if you don't mind , I have a one question. I wonder if you modify your KLD loss like this. It's a bit hard to understand the code, so I wonder if it's different from the existing KLD loss.

KD_loss = nn.KLDivLoss()(F.log_softmax(outputs/T, dim=1), F.softmax(teacher_outputs/T, dim=1)) * (alpha * T * T) + \ F.cross_entropy(outputs, labels) * (1. - alpha)

https://github.com/mary-phuong/multiexit/distillation/blob/faff910ebdcf37c530f5bc419f31d6f587ca1545/main.py#L721

**Comment by mary-phuong** [comment]
Hi,
thanks for getting in touch.

Yes, that's pretty much how I compute the loss.
(See main.py, lines 697--703 in the repo.)

Hope this helps.
mary

**Comment by mary-phuong** [comment]
Here's the link:
https://github.com/mary-phuong/multiexit-distillation/blob/master/main.py#L700
